.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "molecules/molecule-validate-cinder-deploy"]
+	path = molecules/molecule-validate-cinder-deploy
+	url = https://github.com/phongdly/molecule-validate-cinder-deploy
```

Comment:
We should probably move that into rcbops to aid collaboration.

Reply:
Agreed.

Reply:
They may be collapsed in the future, but I wanted to provide a level of indirection while we sort out exactly how the molecule tests should be composed.
@phongdly can you move your repo the rcbops space.

Reply:
@ryan-rs could you add a couple of tests for moleculerize.py ?

Reply:
@johnduarte Sure. I'll hop on it after lunch.